### PR TITLE
[powerapps-component-framework] Update paging API: firstPageNumber, lastPageNumber, pageSize, loadOnlyNewPage

### PR DESCRIPTION
**Pull request by lesyk**
Changes:
- Exposed paging info on the paging object: `firstPageNumber`, `lastPageNumber`, `pageSize`.
- Added a flag based on which page range would change, but only newly loaded page would be loaded (`loadOnlyNewPage`).

Please fill in this template.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by jopursle — CHANGES_REQUESTED** [review]
There appears to be an addition space for the members in Paging.  Format the spacing the same as the other interfaces in the file.

File: types/powerapps-component-framework/componentframework.d.ts
Comment (jopursle) [inline types/powerapps-component-framework/componentframework.d.ts:1948]: Add `@param` to comment

**Comment by jopursle** [comment]
Tests unfortunately should be added for this otherwise it requires DT maintainer approval :(.  Also, the final merge to master should be delayed until the feature this is exposing is available.

**Review by jopursle — APPROVED** [review]

**Comment by lesyk** [comment]
Ready to merge

**Pull request merged**
